controller/api/destination/watcher/endpoints_watcher.go

```diff
@@ -23,11 +25,11 @@ const (
 	// metrics labels
 	service                = "service"
 	namespace              = "namespace"
-	targetGatewayNamespace = "target_gateway_namespace"
-	targetGateway          = "target_gateway"
-	targetCluster          = "target_cluster"
-	targetService          = "target_service"
-	targetServiceNamespace = "target_service_namespace"
+	remoteGatewayNamespace = "remote_gateway_namespace"
```

Comment:
yup, this happened before I opened the PR. I thought it would be a good idea to rebase the changes from master, then I remembered it had changed to main, rebased both and did not notice the new additions until I opened the PR.

Reply:
Yup, should be done now. Sorry it took a bit longer than expected.